addon/components/pop-over.js

```diff
@@ -246,11 +263,13 @@ export default Ember.Component.extend({
   hide() {
     if (this.isDestroyed) { return; }
     set(this, 'active', false);
+    this.removeWindowEvents();
```

Comment:
What about `activate` and `deactivate` methods? Don't we need to add/remove events there as well?